src/pages/basic_data/institution_admin/person_register/PersonTableRow.tsx

```diff
@@ -141,7 +141,7 @@ export const PersonTableRow = ({
   const employmentBaseFieldName = `employments[${employmentIndex}]`;
 
   return (
-    <TableRow>
+    <TableRow onClick={toggleDialog} sx={{ cursor: 'pointer' }}>
```

Comment:
`onClick` burde kanskje fjernes fra linje 172 nå?
Hvis man trykker på selve knappen vil metoden kalles to ganger, slik at den åpnes og lukkes i ett, eller..?

Reply:
Ellers kan du vurdere om vi bare skal fjerne bruken av `IconButton` der også, til noe slik:
```
<Tooltip title={t('common.edit')}>
     <EditIcon />
</Tooltip>
```